test-cypress/integration/explore/class.hierarchy.spec.js

```diff
@@ -191,4 +227,7 @@ describe('Class hierarchy screen validation', () => {
         cy.get('.modal-footer .confirm-btn').click();
         cy.get('.modal').should('not.be.visible');
     }
+
```

Comment:
You left many consecutive blank lines. Can you collapse them to a single blank line where necessary please.

Reply:
Removed blanks in commit beb405691242fb594436842557e670b96b7e6a8b